crates/bevy_ecs/src/query/state.rs

```diff
@@ -140,7 +140,8 @@ impl<Q: WorldQuery, F: ReadOnlyWorldQuery> QueryState<Q, F> {
     }
 
     /// Takes a query for the given [`World`], checks if the given world is the same as the query, and
-    /// generates new archetypes for the given world.
+    /// updates the [`QueryState`] by comparing with the archetypes of the current generation.
+    /// (updates the archetype component access and the matching archetypes/tables)
```

Comment:
```suggestion
    /// updates the [`QueryState`] with any newly-added archetypes.
```

I don't think it's necessary to describe the implementation details of this function.

Reply:
sg!